hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -652,7 +667,7 @@ async def _execute_cancel(self, trading_pair: str, order_id: str) -> str:
                 ex_order_id = await tracked_order.get_exchange_order_id()
 
                 api_params = {
-                    "symbol": await OBDS.exchange_symbol_associated_to_pair(trading_pair),
+                    "symbol": await AscendExAPIOrderBookDataSource.exchange_symbol_associated_to_pair(trading_pair),
```

Comment:
We should be passing the api_factory and the throttler instance in all places where we use the method to translate symbols to trading pairs and trading pairs to symbols